types/utils/global.d.ts

```diff
@@ -24,6 +24,8 @@ interface WebComponentsType {
 declare global {
   interface Window extends Window {
     inert: boolean;
+    recaptchaLoadCallback: () => void;
+    loadScript: (url: string, type?: string) => void;
```

Comment:
Techncially, as both are set at runtime they are `recaptchaLoadCallback?` and `loadScript?`, no?